fendermint/eth/api/src/handlers/http.rs

```diff
@@ -36,6 +36,8 @@ pub async fn handle(
     match call_rpc_str(&state.rpc_server, request).await {
         Ok(result) => {
             tracing::debug!(method, id, result, "RPC call success");
+
+            tracing::debug!("RPC response success: {method:?} - {result:?}");
```

Comment:
Aren't these two logs equivalent? The top version is preferred because it does semantic logging with keywords you can filter on, once we do log aggregation. If you want debug format, you can do one of these:

```rust
tracing::debug!(method, id, ?result, foo = ?bar, "RPC call success");
```

Reply:
Totally, this is just me being sloppy. I forgot to remove that one (which initially was a `println!`). Thanks!